source/base/parameter_handler.cc

```diff
@@ -1219,18 +1219,28 @@ ParameterHandler::set(const std::string &entry_string, const bool new_value)
 
 std::ostream &
 ParameterHandler::print_parameters(std::ostream &    out,
-                                   const OutputStyle style,
-                                   const bool        sort_alphabetical) const
+                                   const OutputStyle style) const
 {
   AssertThrow(out, ExcIO());
 
+  Assert(
```

Comment:
This and the others should be `AssertThrow`. In principle, the output style could be decided at run time, by a parsed string. We want to make sure we assert also in Release mode.